Library/OcSmbiosLib/SmbiosPatch.c

```diff
@@ -1017,26 +1033,45 @@ PatchBootInformation (
 
   @param[in] Table                  Pointer to location containing the current address within the buffer.
   @param[in] Data                   Pointer to tocation containing SMBIOS data.
+  @param[in] AppleSMBIOS            TRUE if Apple SMBIOS present, FALSE otherwise.
```

Comment:
There are only two possible results for a BOOLEAN variable. So, only stating one is enough.

Reply:
I actually looked for examples elsewhere, and the first two I found were like this, so I assumed without looking further that it was the codestyle. But will happily change of course, thanks.